server/models/entity.coffee

```diff
@@ -38,6 +38,7 @@ module.exports = Entity =
     type: 'entity'
     labels: {}
     claims: {}
+    created: Date.now()
```

Comment:
for consistency i would add `updated: Date.now()` field too when entity is created, so that every entity has a `updated` field no matter what

Reply:
done in 3caa775